src/HtmlUtils.tsx

```diff
@@ -399,6 +399,7 @@ export function bodyToHtml(content: IContent, highlights: Optional<string[]>, op
         "mx_EventTile_body": true,
         "mx_EventTile_bigEmoji": emojiBody,
         "markdown-body": isHtmlMessage && !emojiBody,
+        "translate": true,
```

Comment:
Please could you add a comment explaining a bit more about what this does, and why it is necessary? Surely it isn't normally necessary to mark things for translation explicitly?

Do I understand correctly that this is overriding a `notranslate` class somewhere else? Please say that in a comment if so.

Basically: imagine you are a developer reading this code for the first time, without the benefit of having read all of the conversation on https://github.com/element-hq/element-web/issues/25594, and that you are trying to understand what this is for.

Reply:
> Do I understand correctly that this is overriding a notranslate class somewhere else? Please say that in a comment if so.

@richvdh Yes.

The entire application currently has translation disabled using the `notranslate` class placed in the `div#matrixchat` element.
The `translate` class overrides the `notranslate` class that is in this element's ancestor.